zenodo/modules/spam/templates/zenodo_spam/email/spam_user_email.tpl

```diff
@@ -22,6 +22,13 @@
 # as an Intergovernmental Organization or submit itself to any jurisdiction.
 -#}
 
-Our spam protection system has classified your upload as potential spam content. As a preventive measure, we have deactivated your user account.
+Our spam protection system has classified :
+{% if community %}
```

Comment:
Even though it would look a bit ugly, I would inline these in the text above. Otherwise using a `{% set content_text = '...' %}` accordingly is also fine.